cnm/network/network_test.go

```diff
@@ -59,9 +59,10 @@ func TestMain(m *testing.M) {
 		fmt.Printf("Failed to start network plugin %v\n", err)
 		os.Exit(2)
 	}
+	nl := netlink.NewNetlink()
```

Comment:
is this getting compiled in windows? i see netlink specific to linux

Reply:
Yes you can see checks ran on this PR, i created a netlink_windows to help implement the interface for windows too.